aikau/src/main/resources/alfresco/forms/controls/MultiSelectInput.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (C) 2005-2015 Alfresco Software Limited.
+ *
+ * This file is part of Alfresco
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @module alfresco/forms/controls/MultiSelectInput
```

Comment:
It would be nice to get into the habit of adding in some meaningful JSDoc for widgets that we might expect developers to use, I'd quite like to see us get into the habit of using the @example attribute where there might be configuration that is useful to know. In the case of the form controls, most things should be common so it might not be too important to provide an example - but we should at least aim to provide a description of what the form control is.